JSONAPI.EntityFramework.Tests/ActionFilters/JsonApiQueryableActionFilterAttributeTests.cs

```diff
@@ -0,0 +1,1091 @@
+            returnedArray.Length.Should().Be(1);
+            returnedArray[0].Id.Should().Be("370");
+        }
+
+        [TestMethod]
+        public void Filters_by_missing_single_property()
+        {
+            var returnedArray = GetArray<Dummy>("http://api.example.com/dummies?singleField=");
+            returnedArray.Length.Should().Be(0);
+        }
+
+        [TestMethod]
+        public void Filters_by_matching_nullable_single_property()
+        {
+            var returnedArray = GetArray<Dummy>("http://api.example.com/dummies?nullableSingleField=1.3456");
+            returnedArray.Length.Should().Be(1);
+            returnedArray[0].Id.Should().Be("380");
+        }
+
+        [TestMethod]
+        public void Filters_by_missing_nullable_single_property()
+        {
+            var returnedArray = GetArray<Dummy>("http://api.example.com/dummies?nullableSingleField=");
+            returnedArray.Length.Should().Be(48);
+            returnedArray.Any(d => d.Id == "380").Should().BeFalse();
+        }
+
+        #endregion
+
+        #region Double
+
+        [TestMethod]
+        public void Filters_by_matching_double_property()
+        {
+            var returnedArray = GetArray<Dummy>("http://api.example.com/dummies?doubleField=12.3453489012");
+            returnedArray.Length.Should().Be(1);
+            returnedArray[0].Id.Should().Be("390");
+        }
+
+        [TestMethod]
+        public void Filters_by_missing_double_property()
+        {
+            var returnedArray = GetArray<Dummy>("http://api.example.com/dummies?doubleField=");
+            returnedArray.Length.Should().Be(0);
+        }
+
+        [TestMethod]
+        public void Filters_by_matching_nullable_double_property()
+        {
+            var returnedArray = GetArray<Dummy>("http://api.example.com/dummies?nullableDoubleField=34567.1901234");
+            returnedArray.Length.Should().Be(1);
+            returnedArray[0].Id.Should().Be("400");
+        }
+
+        [TestMethod]
+        public void Filters_by_missing_nullable_double_property()
+        {
+            var returnedArray = GetArray<Dummy>("http://api.example.com/dummies?nullableDoubleField=");
+            returnedArray.Length.Should().Be(48);
+            returnedArray.Any(d => d.Id == "400").Should().BeFalse();
+        }
+
+        #endregion
+
+        #region Unknown Type
+
+        [TestMethod]
+        public void Does_not_filter_unknown_type()
+        {
+            var returnedArray = GetArray<Dummy>("http://api.example.com/dummies?unknownTypeField=asdfasd");
+            returnedArray.Length.Should().Be(49);
+        }
+
+        #endregion
+
+        #region Multiple filters
+
+        [TestMethod]
+        public void Ands_together_filters()
+        {
+            var returnedArray = GetArray<Dummy>("http://api.example.com/dummies?stringField=String_value_2&enum_field=3");
```

Comment:
This looks like it has two typos that cause the test to fail, should this be:

    var returnedArray = GetArray<Dummy>("http://api.example.com/dummies?stringField=String value 2&enumField=3");

?